cfme/tests/configure/test_access_control.py

```diff
@@ -1678,7 +1677,34 @@ def test_superadmin_child_tenant_delete_parent_catalog():
         initialEstimate: 1/2h
         startsin: 5.5
     """
-    pass
+    tenant_collection = appliance.collections.tenants
+    root_tenant = tenant_collection.get_root_tenant()
+    catalog_name = fauxfactory.gen_alphanumeric()
+    cat = appliance.collections.catalogs.create(name=catalog_name, description='my catalog')
+    new_tenant = tenant_collection.create(
+        name="tenant{}".format(fauxfactory.gen_alpha(4)),
+        description=fauxfactory.gen_alphanumeric(16),
+        parent=root_tenant)
+
+    group = group_collection.create(description='grp{}'.format(fauxfactory.gen_alphanumeric()),
+                                    role="EvmRole-super_administrator",
+                                    tenant=root_tenant.name + "/" + new_tenant.name)
+    user = new_user(appliance, [group])
+
+    @request.addfinalizer
+    def _delete_user_group_tenant():
+        for item in [user, group, new_tenant]:
+            if item.exists:
+                item.delete()
+
+    assert user.exists
+    try:
+        with user:
+            navigate_to(appliance.server, 'LoggedIn')
```

Comment:
I don't think it's required. please try cat.delete() directly

Reply:
Agree